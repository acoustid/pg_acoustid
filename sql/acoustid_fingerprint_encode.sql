SELECT acoustid_fingerprint_encode('{}v1'::acoustid_fingerprint);
SELECT acoustid_fingerprint_encode('{198178303,198047167,164426175}v1'::acoustid_fingerprint);
