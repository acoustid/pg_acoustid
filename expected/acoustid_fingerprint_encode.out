SELECT acoustid_fingerprint_encode('{}v1'::acoustid_fingerprint);
 acoustid_fingerprint_encode 
-----------------------------
 AQAAAA
(1 row)

SELECT acoustid_fingerprint_encode('{198178303,198047167,164426175}v1'::acoustid_fingerprint);
 acoustid_fingerprint_encode 
-----------------------------
 AQAAA0mSJJGSJMmSCD_uA4AQAQ
(1 row)

